gpu-aware-scheduling/deploy/images/Dockerfile_gpu-extender

```diff
@@ -0,0 +1,35 @@
+#
+# Copyright (c) 2021 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+FROM golang:1.16-alpine as user_builder
+RUN adduser -D -u 10001 gas
+
+FROM golang:1.16-alpine as builder
+ARG DIR=/gpu-aware-scheduling
+WORKDIR $DIR
+COPY . .
+
+RUN mkdir -p /install_root/etc
+COPY --from=user_builder /etc/passwd /install_root/etc/passwd
```

Comment:
Interesting - why are you using a different image to create the user directory?

Reply:
It is needed unfortunately for the COPY. I couldn't figure out a better way and googling it up revealed people were doing just what I was doing also.

Reply:
Plus my end-target was a single-layer image